src/Datadog.Trace.ClrProfiler.Native/clr_helpers.cpp

```diff
@@ -57,14 +57,15 @@ WSTRING GetAssemblyName(const ComPtr<IMetaDataAssemblyImport>& assembly_import,
 FunctionInfo GetFunctionInfo(const ComPtr<IMetaDataImport2>& metadata_import,
                              const mdToken& token) {
   mdToken parent_token = mdTokenNil;
-  WCHAR function_name[kNameMaxSize];
+  WCHAR function_name[kNameMaxSize]{};
```

Comment:
Just curious, what does it change?

Reply:
The `{}` is an initializer syntax. In this case, the empty initializer sets the memory for the array to all zeros instead of leaving whatever data was occupying that space before.

Reply:
Nice, thanks for the explanation :)